src/extension.ts

```diff
@@ -21,9 +21,22 @@ export function activate(context: vscode.ExtensionContext) {
 
     Utility.registerDebugTelemetryListener();
 
-    context.subscriptions.push(vscode.languages.registerCompletionItemProvider([{ language: "json" }, { language: "jsonc" }], new JsonCompletionItemProvider(), "\"", ".", ":"));
-
-    const outputChannel: vscode.OutputChannel = vscode.window.createOutputChannel(Constants.edgeChannel);
+    const configIntelliSenseProvider = new ConfigIntelliSenseProvider();
```

Comment:
Is there any reason we put three kinds of provider into one class?

Reply:
I have split it into corresponding classes.